docs/en/operation/Server-Logging.md

```diff
@@ -0,0 +1,162 @@
+---
+layout: global
+title: Server Logging
+nickname: Server Logging
+group: Operations
+priority: 0
+---
+
+* Table of Contents
+{:toc}
+
+## Introduction
+
+This page summarizes Alluxio's logging system for servers, such as the master and worker
+processes. We include tips for modifying Alluxio's log4j properties file to best suit
+deployment needs. If you are a user of Alluxio looking for logging information about clients
+that are utilizing the Alluxio API, we recommend looking at the
+[client logging documentation]({{site.baseurl}}{% link en/advanced/Client-Logging.md %})
+
+Alluxio's logging behavior can be fully configured through the `log4j.properties` file found in the
+`conf` folder.
+
+## Log Location
+
+### Default Location
+
+By default Alluxio processes' log files can be found under `${ALLUXIO_HOME}/logs/`.
+
+### Configuring the Log Location 
+
+The location of the logs is determined by the `alluxio.logs.dir` property. This is a JVM
+property which cannot be set in the `alluxio-site.properties` file. See the
+[configuration settings page]({{site.baseurl}}{% link en/advanced/Configuration-Settings.md %}#configuration-sources)
+for more information on how to set JVM properties for Alluxio.
+
+Each Alluxio process (master, worker, FUSE, proxy) will log to a separate file within the
+`alluxio.logs.dir` directory
+
+## Log Levels
+
+### Modifying Logging with `log4j.properties`
+
+You can modify the `log4j.properties` found under the `${ALLUXIO_HOME}/conf/log4j.properties` to modify logging levels.
```

Comment:
Should we give an example of how to change the logging to DEBUG?

Reply:
I agree. I will add a paragraph with an example